deployment/deploy-live.sh

```diff
@@ -0,0 +1,9 @@
+
+cd /var/www/weatherstations_frontend
+git stash
+git pull
+php artisan migrate
+composer install
+npm install
+npm run dev
```

Comment:
I'd do `npm run prod` for production / live versions. It does a few extra things like minifying the code to make it more efficient for production.